packages/design-system/package.json

```diff
@@ -39,8 +39,8 @@
     "docs": "pnpm typedoc --logLevel Error --entryPoints src/index.ts --out documentation/api --readme none --disableSources --plugin typedoc-plugin-markdown",
     "index": "pnpm zx ../../scripts/indexer.mjs packages/design-system --barrels",
     "lint": "pnpm biome lint ./",
-    "preview": "pnpm ladle serve",
-    "test": "pnpm vitest --dir=src"
+    "test": "pnpm vitest --dir=src",
+    "storybook": "storybook dev -p 6006"
```

Comment:
I would prefer to keep the script as `preview`

Reply:
updated: https://github.com/gohypergiant/standard-toolkit/pull/167/commits/c8c00012ed9518463c70270bbedc2fdd8c37f677

alternatively, wdyt about calling it `dev`?